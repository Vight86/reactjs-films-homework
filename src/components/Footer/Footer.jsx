import React from 'react';
import Logo from '../Logo/index';
import Signature from '../Signature/index';
import styles from './Footer.scss';

const Footer = () => (
  <footer className={styles.footer}>
    <Logo className="secondary">Films</Logo>
    <p className={styles.footer__copyRight}>
      Copyright Â© 2018 FILMS.
      {' '}
      <Signature name="by Vova Pekun" />
    </p>
  </footer>
);

export default Footer;
