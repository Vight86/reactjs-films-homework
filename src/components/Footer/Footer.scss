@import '../../utils/variables';

.footer {
  padding-top: 8em;
  padding-bottom: 5.5em;
  background-color: $color-light;
  text-align: center;

  &__copyRight {
    margin-top: 3em;
    color: $text-tertiary;
    font-family: $font-primary;
    font-size: 2.4em;
    opacity: 0.4;
  }
}
