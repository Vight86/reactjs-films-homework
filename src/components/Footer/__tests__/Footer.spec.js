import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../Footer';

let renderer;
let result;

beforeEach(() => {
  renderer = new ShallowRenderer();
  renderer.render(<Footer />);
  result = renderer.getRenderOutput();
});

afterEach(() => {
  renderer = null;
  result = null;
});

describe('render Footer component', () => {
  it('render correctly', () => {
    expect(result).toMatchSnapshot();
  });
});
