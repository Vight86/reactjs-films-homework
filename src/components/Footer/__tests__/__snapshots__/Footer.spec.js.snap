// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer component render correctly 1`] = `
<footer
  className="footer"
>
  <Logo
    className="secondary"
  >
    Films
  </Logo>
  <p
    className="footer__copyRight"
  >
    Copyright Â© 2018 FILMS.
     
    <Signature
      name="by Vova Pekun"
    />
  </p>
</footer>
`;
